src/colorbar/index.mjs

```diff
@@ -0,0 +1,94 @@
+import React from 'react'
+import { Box, Flex } from 'theme-ui'
+import useColormap from '../use-colormap'
+
+import Gradient from './gradient'
+import Label from './label'
+
+const ColorbarInner = ({
+  sx,
+  label,
+  colormap,
+  clim,
+  units,
+  format,
+  horizontal,
+}) => {
+  const climMin = (
+    <Box
+      sx={{
+        fontFamily: 'mono',
```

Comment:
might be slightly cleaner to pull this and the subsequent 3 lines out into a variable, because we apply the same set twice